src/core/expr/fexpr_nth.cc

```diff
@@ -106,20 +107,21 @@ class FExpr_Nth : public FExpr_Func {
 static py::oobj pyfn_nth(const py::XArgs& args) {
   auto cols = args[0].to_oobj();
   auto n = args[1].to<int32_t>(0);
-  auto skipna = args[2].to<bool>(false);
-
-  if (skipna) {
-    return PyFExpr::make(new FExpr_Nth<true>(as_fexpr(cols), n));
-  } else {
-    return PyFExpr::make(new FExpr_Nth<false>(as_fexpr(cols), n));
+  auto dropna = args[2].to<std::string>("None");
+  bool dropna_check = dropna == "any" || dropna == "all" || dropna == "None";
+  if (!dropna_check) {
+    throw ValueError() << "Parameter `dropna` in nth() should be "
+        "either `None`, `any`, or `all`";
```

Comment:
For `None` we don't need it to be a string. `None` could be a Python's `None`. We can also keep the name as `skipna`. Not sure why pandas has both `skipna` and `dropna` for the same purpose.